app/Controllers/Http/ServiceController.ts

```diff
@@ -104,38 +100,59 @@ class ServiceController {
         workspaces: [],
         ...settings,
         iconUrl: settings.iconId
-          ? `${Env.get('APP_URL')}/v1/icon/${settings.iconId}`
-          : null,
+          ? `${url}/v1/icon/${settings.iconId}`
+          : // eslint-disable-next-line unicorn/no-null
```

Comment:
Feel free to disable that lint rule directly in eslint config, I always do that for all projects where I use eslint-plugin-unicorn, guess I forgot here when I implemented it.